pkg/trace/api/api.go

```diff
@@ -77,8 +78,10 @@ type HTTPReceiver struct {
 	debug               bool
 	rateLimiterResponse int // HTTP status code when refusing
 
-	wg   sync.WaitGroup // waits for all requests to be processed
-	exit chan struct{}
+	appsecHandler http.Handler
```

Comment:
Can you move this field into the bigger group above please? The one with `out`, `conf`, etc.

Reply:
Fixed